### PR TITLE
[DJFS-27] Develop an E-commerce Store Database

### PR DESCRIPTION
**Pull request by pelumi454**
## Task: `Name of given task`
- `Changes you made to given task`

### CODE REVIEW

File: submissions/amooChristianah/database/init.sql [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+ VALUES('shoes'),
+ ('colognes'),
+ ('sunglasses'),
+ ('clothing');
+ 
+ SELECT * FROM categories;
+ 
+ -- 3. PRODUCT TABLE
+ -- creating table
+    CREATE TABLE product(
```
Comment (adaugochi) [inline submissions/amooChristianah/database/init.sql:88]: By convention, table names should be pluralized

File: submissions/amooChristianah/database/init.sql [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+SELECT * FROM orders;
+-- 7. ORDER ITEM TABLE
+-- creating table
+CREATE TABLE order_items(
+ id INT NOT NULL auto_increment,
+order_id INT NOT NULL,
+product_id INT NOT NULL,
+quantity INT NOT NULL,
+unit_prices INT NOT NULL,
```
Comment (adaugochi) [inline submissions/amooChristianah/database/init.sql:180]: use `DECIMAL` for money

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/amooChristianah/database/init.sql (hunk 1 above)
Comment (adaugochi) [reply submissions/amooChristianah/database/init.sql:88]: By convention, table names should be pluralized

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by pelumi454** [comment]
@adaugochi kindly review my PR

File: submissions/amooChristianah/database/TransportationCompany.sql [hunk 3]
```diff
@@ -0,0 +1,44 @@
+create DATABASE TRANSPORTATION;
```
Comment (adaugochi) [inline submissions/amooChristianah/database/TransportationCompany.sql:1]: This supposes to be for your next task. Kindly remove it

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/amooChristianah/database/TransportationCompany.sql (hunk 3 above)
Comment (pelumi454) [reply submissions/amooChristianah/database/TransportationCompany.sql]: done. kindly review

**Review by adaugochi — APPROVED** [review]

**Pull request merged**